### ISSUE TITLE
[Support]: Nvidia Jetson Support Update

### ISSUE THREAD

**Issue by AndBobsYourUncle** [blakeblackshear/frigate#8907]
### Describe the problem you are having

It seems that the current Jetson support does not work with devices that have Jetpack 5.1.2 or newer installed.

This is because `nvbuf_utils` was completely removed as a shared library. The custom jetpack-ffmpeg that is built for the Jetson Docker image is linking `nvbuf_utils` to the final patched ffmpeg. However, if this library is missing on the device (because it is running Jetpack 5.1.2 or newer), then trying to start up the Frigate Docker container will result in Frigate not spinning up successfully, with an error output:
```
2023-12-09 15:30:46.148383273  [INFO] The go2rtc service exited with code 1 (by signal 0)
2023-12-09 15:30:47.082353197  Traceback (most recent call last):
2023-12-09 15:30:47.082363533    File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
2023-12-09 15:30:47.082528335      return _run_code(code, main_globals, None,
2023-12-09 15:30:47.082532591    File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
2023-12-09 15:30:47.082599727      exec(code, run_globals)
2023-12-09 15:30:47.082603311    File "/opt/frigate/frigate/__main__.py", line 6, in <module>
2023-12-09 15:30:47.082662960      from frigate.app import FrigateApp
2023-12-09 15:30:47.082683312    File "/opt/frigate/frigate/app.py", line 20, in <module>
2023-12-09 15:30:47.082739377      from frigate.comms.dispatcher import Communicator, Dispatcher
2023-12-09 15:30:47.082743313    File "/opt/frigate/frigate/comms/dispatcher.py", line 7, in <module>
2023-12-09 15:30:47.082777969      from frigate.config import BirdseyeModeEnum, FrigateConfig
2023-12-09 15:30:47.082779921    File "/opt/frigate/frigate/config.py", line 28, in <module>
2023-12-09 15:30:47.082852562      from frigate.ffmpeg_presets import (
2023-12-09 15:30:47.082856946    File "/opt/frigate/frigate/ffmpeg_presets.py", line 47, in <module>
2023-12-09 15:30:47.082901682      if int(os.getenv("LIBAVFORMAT_VERSION_MAJOR", "59")) >= 59
2023-12-09 15:30:47.082914578  ValueError: invalid literal for int() with base 10: ''
ffmpeg: error while loading shared libraries: libnvbuf_utils.so.1.0.0: cannot open shared object file: No such file or directory
```

This is because in Jetpack 5.1.2, `libnvbuf_utils.so.1.0.0` is completely missing, and is not installable.

To replicate this, just try starting up Frigate on an Nvidia Jetson device running Jetpack 5.1.2 or newer:
```
docker run \
  --rm \
  --name frigate \
  --mount type=tmpfs,target=/tmp/cache,tmpfs-size=1000000000 \
  --gpus=all \
  --shm-size=64m \
  -v /home/somebody/frigate/recordings/media/frigate \
  -v /home/somebody/frigate/config:/config \
  -v /etc/localtime:/etc/localtime:ro \
  -e FRIGATE_RTSP_PASSWORD='password' \
  -e YOLO_MODELS=yolov7-320-dla \
  -p 5000:5000 \
  -p 8554:8554 \
  -p 8555:8555/tcp \
  -p 8555:8555/udp \
  ghcr.io/blakeblackshear/frigate:0.13.0-beta6-tensorrt-jp5
```

The fix:
https://github.com/madsciencetist/jetson-ffmpeg/pull/3

Once I built the Jetpack 5 Frigate Docker image from my own fork of Frigate that pointed to my own fork of `jetson-ffmpeg` (that has the changes from the PR above), I was not only able to start up Frigate, but I had detections working as well.

The change above in the PR also doesn't seem like it would cause any issues for someone using Jetpack 5.1.1.

### Version

0.13.0-beta6

### Frigate config file

```yaml
mqtt:
  enabled: False

ffmpeg:
  hwaccel_args: preset-jetson-h264

cameras:
  front_door: # <------ Name the camera
    ffmpeg:
      inputs:
        - path: rtsp://192.168.1.20:8554/unicast # <----- The stream you want to use for detection
          roles:
            - detect
    detect:
      enabled: True # <---- disable detection until you have a working camera feed
      fps: 10
      width: 1024 # <---- update for your camera's resolution
      height: 768 # <---- update for your camera's resolution

objects:
  # Optional: list of objects to track from labelmap.txt (default: shown below)
  track:
    - person

detectors:
  tensorrt:
    type: tensorrt
    device: 0 #This is the default, select the first GPU

model:
  path: /config/model_cache/tensorrt/yolov7-320-dla.trt
  input_tensor: nchw
  input_pixel_format: rgb
  width: 320
  height: 320
```

### Relevant log output

```shell
2023-12-09 15:30:46.148383273  [INFO] The go2rtc service exited with code 1 (by signal 0)
2023-12-09 15:30:47.082353197  Traceback (most recent call last):
2023-12-09 15:30:47.082363533    File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
2023-12-09 15:30:47.082528335      return _run_code(code, main_globals, None,
2023-12-09 15:30:47.082532591    File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
2023-12-09 15:30:47.082599727      exec(code, run_globals)
2023-12-09 15:30:47.082603311    File "/opt/frigate/frigate/__main__.py", line 6, in <module>
2023-12-09 15:30:47.082662960      from frigate.app import FrigateApp
2023-12-09 15:30:47.082683312    File "/opt/frigate/frigate/app.py", line 20, in <module>
2023-12-09 15:30:47.082739377      from frigate.comms.dispatcher import Communicator, Dispatcher
2023-12-09 15:30:47.082743313    File "/opt/frigate/frigate/comms/dispatcher.py", line 7, in <module>
2023-12-09 15:30:47.082777969      from frigate.config import BirdseyeModeEnum, FrigateConfig
2023-12-09 15:30:47.082779921    File "/opt/frigate/frigate/config.py", line 28, in <module>
2023-12-09 15:30:47.082852562      from frigate.ffmpeg_presets import (
2023-12-09 15:30:47.082856946    File "/opt/frigate/frigate/ffmpeg_presets.py", line 47, in <module>
2023-12-09 15:30:47.082901682      if int(os.getenv("LIBAVFORMAT_VERSION_MAJOR", "59")) >= 59
2023-12-09 15:30:47.082914578  ValueError: invalid literal for int() with base 10: ''
ffmpeg: error while loading shared libraries: libnvbuf_utils.so.1.0.0: cannot open shared object file: No such file or directory
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

Other

### Network connection

Wireless

### Camera make and model

Custom RTSP Raspberry PI Camera

**Comment by AndBobsYourUncle** [comment]
@madsciencetist

Since I believe that this was you recent addition to the 0.13 beta, and you are also the maintainer of the jetson-ffmpeg repo that is used.

**Comment by AndBobsYourUncle** [comment]
I have also just opened a PR that fixes this issue:
https://github.com/blakeblackshear/frigate/pull/8914

**Comment by madsciencetist** [comment]
Fixed by #8914
